guides/source/getting_started.md

```diff
@@ -7,1150 +7,1548 @@ This guide covers getting up and running with Ruby on Rails.
 
 After reading this guide, you will know:
 
-* How to install Rails, create a new Rails application, and connect your
-  application to a database.
+* How to install Rails, create a new Rails application, and connect your application to a database.
 * The general layout of a Rails application.
 * The basic principles of MVC (Model, View, Controller) and RESTful design.
 * How to quickly generate the starting pieces of a Rails application.
-
+* How to deploy your app to production using Kamal.
```

Comment:
Not sure "How to deploy your app to production" can be part of Getting Started guide.